corehq/apps/user_importer/helpers.py

```diff
@@ -161,9 +161,13 @@ def update_user_data(self, data, uncategorized_data, profile_name, profiles_by_n
             raise UserUploadError(_("You cannot set {} directly").format(PROFILE_SLUG))
         if profile_name:
             profile_id = profiles_by_name[profile_name].pk
+        elif profile_name == '':
+            profile_id = ''
```

Comment:
I think this should be
```suggestion
            profile_id = None
```

Reply:
Hmm.  It doesn't look super clear one way or the other from `user_data.py`, which I think is probably an artifact of the history of this field (it was a Django PK stored in and unschema'd JSON dict).  This is stored on `SQLUserData` as a `ForeignKey` field, which I think expects either an integer or `None`.  I just played with it a bit and got it to fail in some situations, but not through the accessor, so I think I'm missing something.  Still, it's probably better to be certain we can't set that as an empty string, so I just pushed a commit to this branch ensuring as much (sorry to hijack the branch!).  I think that makes my complaint here moot.